### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by karayamanemre**
In this pull request of our portfolio website, we will implement the mobile menu using our recently acquired JavaScript knowledge.

We did the following below;
- Used basic JavaScript syntax.
- Used JavaScript to manipulate DOM elements.
- Used JavaScript events.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -25,9 +24,9 @@
               <span class="line"></span>
             </button>
             <ul class="nav-menu flex">
-              <li class="item flex"><a href="#work">Portfolio</a></li>
-              <li class="item flex"><a href="#about">About</a></li>
-              <li class="item flex"><a href="#contact">Contact</a></li>
+              <li class="item flex"><a href="#work" class="nav-link">Portfolio</a></li>
+              <li class="item flex"><a href="#about" class="nav-link">About</a></li>
+              <li class="item flex"><a href="#contact" class="nav-link">Contact</a></li>
```
Comment (phelian23) [inline index.html:29]: - [x] Good job on working on the mobile menu requirements, everything works as expected. However, one of the goals of this project is to learn how to follow a design template. kindly make the width of the mobile menu full as it is in the design template 😉.

**Review by phelian23 — CHANGES_REQUESTED** [review]
Hi @karayamanemre,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ Following GitHub Flow
✔️ Following requirements
✔️ Descriptive PR Title
✔️ Descriptive PR Summary

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (UsirusenIkon) [reply index.html:29]: I just implemented your review, but looking at UI specifically, this doesn't look good, especially on tablet devices

**Review by ShahierNashaat — APPROVED** [review]
Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 💯 

- Correct use of GitFlow ✔️
- Descriptive commit messages ✔️
- Professional looking pull request ✔️
- Your readme looks professional ✔️
- Good overall design ✔️
- Your code is clean and well documented ✔️

# Approved :ok_hand: :ok_hand: :ok_hand:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**